### PR TITLE
EngineBase: add missing lang name for 'frm'

### PR DESCRIPTION
**Pull request by MusikAnimal**
Also remove 'equ' as it's been removed from langs.json

Bug: T282760

### CODE REVIEW

File: src/Engine/EngineBase.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ abstract class EngineBase
         'de-frk' => 'Deutsch (Fraktur)',
         'enm' => 'Middle English (1100-1500)',
         'es-old' => 'español (viejo)',
-        'equ' => 'Math / equation detection module',
+        'frm' => 'français moyen (1400-1600)',
```
Comment (Daimona) [inline src/Engine/EngineBase.php]: I think this should be "moyen français", according to https://fr.wikipedia.org/wiki/Moyen_fran%C3%A7ais

Comment (Daimona) [reply src/Engine/EngineBase.php]: I mean, "moyen français (1400-1600)", the swap is the important thing.

Comment (MusikAnimal) [reply src/Engine/EngineBase.php]: Done

**Comment by Daimona** [comment]
Thank you! Tomorrow I might follow-up with a test that ensures that every language has an entry where it needs to.

**Comment by MusikAnimal** [comment]
> Thank you! Tomorrow I might follow-up with a test that ensures that every language has an entry where it needs to.

Good idea! I just submitted #44

**Comment by Daimona** [comment]
> > Thank you! Tomorrow I might follow-up with a test that ensures that every language has an entry where it needs to.
> 
> Good idea! I just submitted #44

Thank you!

**Pull request merged**